src/menu/host_interface.c

```diff
@@ -113,7 +114,29 @@ void main_menu_host_interface(engine_ctx_t *ctx,
       break;
     }
     case 2: {
-      btc_app_main(usb_evt);
+      // TODO: instantiate & configure Bitcoin chain
```

Comment:
Are these TODO's required?

Reply:
The Bitcoin one can be removed.

Reply:
removed one comment and corrected the others